app/Repositories/Repository.php

```diff
@@ -65,7 +66,11 @@ public function update($model, $data) {
     }
 
     public function updateOrCreate($model, $target,$data) {
+        dd($model);
         $model->updateOrCreate($target, $data);
+        if ($data->payment_type_id == 2) {
```

Comment:
whats this?

Reply:
My bad. I forgot to remove from repository